packages/apps-v5/src/commands/releases/index.js

```diff
@@ -1,171 +0,0 @@
-'use strict'
-
-const cli = require('heroku-cli-util')
-const stripAnsi = require('strip-ansi')
-
-async function run(context, heroku) {
-  const statusHelper = require('../../status_helper')
-  const time = require('../../time')
-  const {truncate} = require('lodash')
-
-  let optimizationWidth = 0
-
-  let descriptionWithStatus = function (d, r) {
-    const width = () => process.stdout.columns > 80 ? process.stdout.columns : 80
-    const trunc = (s, l) => {
-      if (process.stdout.isTTY) {
-        return truncate(s, {length: width() - (optimizationWidth + l), omission: '…'})
-      }
-
-      return s
-    }
-
-    const status = statusHelper.description(r, runningRelease, runningSlug)
-    if (status) {
-      const sc = cli.color[statusHelper.color(r.status)](status)
-      return trunc(d, status.length + 1) + ' ' + sc
-    }
-
-    return trunc(d, 0)
-  }
-
-  let url = `/apps/${context.app}/releases`
-  if (context.flags.extended) url += '?extended=true'
-  let releases = await heroku.request({
-    path: url,
-    partial: true,
-    headers: {
-      Range: `version ..; max=${context.flags.num || 15}, order=desc`,
-    },
-  })
-
-  let header = `${context.app} Releases`
-  // eslint-disable-next-line unicorn/prefer-array-find
-  let currentRelease = releases.filter(r => r.current === true)[0]
-  if (currentRelease) {
-    header += ' - ' + cli.color.blue(`Current: v${currentRelease.version}`)
-  }
-
-  let runningRelease = releases.filter(r => r.status === 'pending').slice(-1)[0]
-  let runningSlug
-  if (runningRelease && runningRelease.slug) {
-    runningSlug = await heroku.get(`/apps/${context.app}/slugs/${runningRelease.slug.id}`)
-  }
```

Comment:
This entire block was deleted because runningSlug is never used. All unit tests had to have nock mocks updated as well.